### PR TITLE
Create telephone database structure

### PR DESCRIPTION
**Pull request by izumizawa**
Created telephone table structure:
telephone_id = *telefone_id / UUID / PK
client_id = *cliente_id / UUID / FK da tabela cliente
telephone_id_type = *tipo_telefone_id / int / FK tipo_telefone
number = *numero / varchar(15)
ddd = *ddd / varchar(3)

Created telephone id type table structure:
telephone_id_type = *tipo_telefone_id / int / PK
telephone_type = *tipo_telefone / varchar(20)

Inserted in telephone id type:
1 = landline
2 = mobile

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -1,4 +1,83 @@
...
+
+  -  changeSet:
+       id:  Adding telephone type table
+       author:  izumizawa
+       changes:
+         -  createTable:
+              tableName: telephone_type
+              columns:
+                - column:
+                    name: telephone_id_type
```
Comment (rgueltamatera) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: maybe "telefone_type_id" is better here :)

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -1,4 +1,83 @@
...
+                    type: int
+                    constraints:
```
Comment (rgueltamatera) [inline src/main/resources/db/changelog/db.changelog-master.yaml:88]: Can you please add a PK name here? like "telefone_type_pk".
You can also remove the nullable constraint as it's is redundant, PKs can't be null by default :)

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 3]
```diff
@@ -1,4 +1,83 @@
...
+       id:  Adding telephone table
+       author:  izumizawa
+       changes:
+         -  createTable:
+              tableName: telephone
+              columns:
+                -  column:
+                     name: telephone_id
+                     type: uuid
+                     constraints:
```
Comment (rgueltamatera) [inline src/main/resources/db/changelog/db.changelog-master.yaml:105]: we are missing PK name here as well :)

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -1,4 +1,83 @@
...
+                -  column:
+                     name: client_id
+                     type: uuid
+                     constraints:
+                       nullable: false
+                -  column:
+                     name: telephone_id_type
+                     type: int
+                     constraints:
+                       foreignKeyName: telephone_id_type
```
Comment (rgueltamatera) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: maybe add "_fk" to FK name so we know this constraint is a FK, like "telefone_type_fk"

**Review by gabrieloswaldo — APPROVED** [review]
It's all ok! I think you just need to solve Robin's comments, and then you can merge it :)

**Review by gdseabra — APPROVED** [review]

**Pull request merged**